### PR TITLE
Provide path to plugin in repository

### PR DESCRIPTION
**Pull request by Perlence**
This pull request is based upon issue #11. It adds optional flag for URL and an optional argument `--path PATH` that specifies root of the plugin in repository to be loaded by fundle.

The usage of `fundle plugin` is now:

```
fundle plugin NAME [[--url] URL] [--path PATH]
```

### CODE REVIEW

File: functions/fundle.fish [hunk 1]
```diff
@@ -254,19 +271,40 @@ end
...
-		case 2
-			set plugin_url $argv[2]
+	set -l plugin_path "."
+	set -l argv_count (count $argv)
+	set -l skip_next true
+	if test $argv_count -eq 0 -o -z "$argv"
+		echo "usage: fundle plugin NAME [[--url] URL] [--path PATH]"
+		return 1
+	else if test $argv_count -gt 1
+		set -l state "start"
```
Comment (danhper) [inline functions/fundle.fish:281]: This can be removed.

**Comment by danhper** [comment]
Great job, thank you very much!
It looks very nice, I'll release a new version including this feature.

**Comment by danhper** [comment]
@Perlence I just updated the documentation and released v0.4.0 with your changes, thanks again!

**Comment by Perlence** [comment]
@tuvistavie You are welcome :+1:

**Comment by Perlence** [comment]
@tuvistavie Just to clarify, `--url` bit is not required, I did it for backward compatibility. Did you intend to drop this compatibility in future versions when you were writing docs?

**Comment by danhper** [comment]
@Perlence Yes, thanks I noticed it was not required but I think I will drop it in future versions, so I changed the docs to use only the usage with `--url`.
I will probably add a deprecation notice for the second argument in the next release.

**Pull request merged**